setup.cfg

```diff
@@ -49,6 +49,7 @@ install_requires =
     holidays >= 0.13
     numpy >= 1.21.0
     pandas >= 1.3.0
+    packaging >= 20.0
```

Comment:
It looks like `distributed` has `packaging >= 20.0` so I put this as the minimal version. It's also the release of `packaging` that supports Python3.8